app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,28 @@
+class FriendshipsController < ApplicationController
+  before_action :set_friendship, only: [:destroy]
+
+  def create
+    @friendship = Friendship.new
+    request = FriendRequest.find(params[:friend_request_id])
```

Comment:
why does request not need to be an instance variable here?

Reply:
What I'm doing here is when you accept a friend request, we are creating new empty Friendship and in the params we have friend request id so we are finding a friend request with params friend request id then updating the friendship